src/components/styles/ProjectPreview.styled.ts

```diff
@@ -0,0 +1,54 @@
+import styled, { keyframes, FontType, css } from 'styled-components'
+import { fontTypeCss } from '../../styles/index.styled'
+
+const FadeIn = keyframes`
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+`
+
+export const PreviewText = styled.div<{ hasAnimation?: boolean; fontType: FontType }>`
+  ${fontTypeCss}
+  color: ${(props) => props.theme.palette.common.white};
+  animation-name: none;
```

Comment:
not necessary since there is no animation if hasAnimation is not true

Reply:
92c245dd8781d90e0d89f8cd2bad5f03c6ecf6e0